CMSgov/mint-app
# Easi 2469/interative tag

# EASI-2469

## Changes and Description

- Created new landing pages and routes for each of the tags: `Getting Started` and `IT Implementation`
- Add new `HelpTag` to `ArticleCard` file. Similar to EASI's help tag
- add new custom css: `.width-fit-content { width: fit-content } `
- 

## Acceptance Criteria

- Make existing tags clickable
- Each tag should have its own page/view displaying all articles with that tag
- Articles on the tag view should not display that same tag on the article - all other areas should display the tag on the article
- Unit tests



## How to test this change

1. Navigate to `Help and Knowledge Center`
2. Click on tag
3. User should be navigated to a landing page that has articles that only have `Getting STarted` or `IT Implmentation` tags and matches figma 


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
